solver/src/solver/http_solver/settlement.rs

```diff
@@ -139,67 +125,25 @@ fn match_prepared_and_settled_orders(
 }
 
 fn match_prepared_and_settled_amms(
```

Comment:
Other side of the :meat_on_bone:, we no longer need to deal with split liquidity.